torch/onnx/symbolic_opset11.py

```diff
@@ -279,6 +279,19 @@ def append(g, self, tensor):
     return g.op("SequenceInsert", self, tensor)
 
 
+def add(g, self, other, alpha=None):
+    if isinstance(self.type(), torch._C.Type) and self.type().isSubtypeOf(torch._C.ListType.ofTensors()):
```

Comment:
Try _is_value and _is_tensor_list from symbolic helper.

Reply:
Thanks!